include/wx/notifmsg.h

```diff
@@ -81,6 +81,15 @@ class WXDLLIMPEXP_CORE wxNotificationMessageBase : public wxEvtHandler
         Timeout_Never = 0   // notification will never time out
     };
 
+    // reasons for dismissal, posted with wxEVT_NOTIFICATION_MESSAGE_DISMISSED events
+    enum class DismissalReason
+    {
+        Unknown,
+        DismissedByUser,
+        DismissedByApp,
```

Comment:
Using `DismissalReason::DismissedByUser` in the code looks a bit heavy, maybe we should call this just `ByUser`/`ByApp`?

Reply:
Sounds good.